rnndb/graph/g80_pgraph/tpc.xml

```diff
@@ -42,8 +42,8 @@
 		<bitfield name="DST2D_STORAGE_TYPE_MISMATCH" pos="8" />
 		<bitfield name="ZETA_STORAGE_TYPE_MISMATCH" pos="9" />
 		<bitfield name="RT_STORAGE_TYPE_MISMATCH" pos="10" />
-		<bitfield name="DST2D_LINEAR_MISMATCH" pos="11" />
```

Comment:
Sounds weird this way...

This error triggers when ROP/TEX is set for blocklinear mode, but finds pitch-format memory, or the other way around. PITCH_MISMATCH instead sounds like some sort of error in setting up the pitch (as in, distance between lines).

Perhaps call it MODE_MISMATCH?